cmd/containeragent/unit/agent.go

```diff
@@ -187,8 +186,10 @@ func (c *containerUnitAgent) Init(args []string) error {
 	if err := c.ensureToolSymlinks(srcDir, dataDir, unitTag); err != nil {
 		return errors.Annotate(err, "ensuring agent tool symlinks")
 	}
-
-	c.containerNames = strings.Split(c.environment.Getenv("JUJU_CONTAINER_NAMES"), ",")
+	containerNames := c.environment.Getenv("JUJU_CONTAINER_NAMES")
```

Comment:
We also do this in `cmd/containeragent/initialize/command.go`.
I don't think it is used after that, but probably worth adding this check there also just in case